### ISSUE TITLE
Add a tiny warning when Gem not found and disable_system_gems is enabled

### ISSUE THREAD

**Issue by merbjedi** [wycats/bundler#110]
Minor suggestion:

Was banging my head against the wall the other day when gems installed locally weren't being installed via "gem bundle" command. Turns out I had disable_system_gems on. I know, dumb error, but it would be nice if it gave a more detailed warning after it cant find a gem of where it tried to look.

**Comment by carlhuda** [comment]
Are you saying that gem bundle is not pulling gems from the local system gems when disable_system_gems is on?

**Comment by merbjedi** [comment]
Yup. I'd always thought disable_system_gems was a runtime thing so it wouldnt accidently require gems from the system path. Had forgotten that it applied to the bundle command as well.
